cmd/migrate/main.go

```diff
@@ -16,7 +16,7 @@ import (
 
 func init() {
 	pflag.Parse()
-	viper.BindPFlags(pflag.CommandLine)
+	_ = viper.BindPFlags(pflag.CommandLine)
```

Comment:
Wouldn't it be better to check this return value than to ignore it?

Reply:
Yes, fixed it as discussed